components/gfx/platform/freetype/font.rs

```diff
@@ -447,8 +447,8 @@ impl FreeTypeFaceHelpers for FT_Face {
         let requested_size = (requested_size.to_f64_px() * 64.0) as i64;
         let get_size_at_index = |index| unsafe {
             (
-                (*(*self).available_sizes.offset(index as isize)).x_ppem as i64,
-                (*(*self).available_sizes.offset(index as isize)).y_ppem as i64,
```

Comment:
This is necessary for the build to work on Android. Please undo this change. We'll have to make clippy happy in some other way.